### ISSUE TITLE
WARNING:tensorflow:AutoGraph is not available in this environment: functions lack code information.

### ISSUE THREAD

**Issue by jhcuarta** [CorvusCodex/LotteryAi#4]
Hi
I'm using the executable application .exe and I get the following error "WARNING:tensorflow:AutoGraph is not available in this environment: functions lack code information. This is typical of some environments like the interactive Python shell. See https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/autograph/g3doc/reference/limitations.md#access-to-source-code for more information." I'm concern cause it says functions lack code information, reason why I think I'm getting high values for val_loss, for instance:

Epoch 5/100
1/5 [=====>........................] - ETA: 0s - loss: 538.5345 - accuracy: 1.00
5/5 [==============================] - ETA: 0s - loss: 673.7310 - accuracy: 1.00
5/5 [==============================] - 0s 40ms/step - loss: 673.7310 - accuracy:
 1.0000 - val_loss: 802.8627 - val_accuracy: 1.0000
Epoch 6/100
1/5 [=====>........................] - ETA: 0s - loss: 800.9542 - accuracy: 1.00
4/5 [=======================>......] - ETA: 0s - loss: 797.1685 - accuracy: 1.00
5/5 [==============================] - 0s 33ms/step - loss: 776.6075 - accuracy:
 1.0000 - val_loss: 823.5990 - val_accuracy: 1.0000

I've manage to code a script as a loop for executing the application 100 times, and the lower value I got was close to 300, which I guess it is still high, and it is maybe due to autograph error on logging to source code. I even change to Ununtu 22.04 OS, so I can easily manipulate the python scripts, nevertheless no success; I also built tensorflow from source and no success either. I edited autograph module according directions from webpage provided and didn't work, or I don't know what I missed; but I'm quite confused because despite the perfect accurracy there's a high value loss.

Thanks in advance

**Comment by jhcuarta** [comment]
Hi
I was wondering if you could help me out with this error, I haven't been able to play lotto with a favorable confidence criterion due to the warning. I'm really disappointed since i bought the windows executable and no support at all

**Comment by CorvusCodex** [comment]
This is typical of some environments like the interactive Python shell. Does not affect ai training. Support comes only with the membership.
